Update active-directory-ds-admin-guide-join-rhel-linux-vm.md

paulomarquesc opened the pull request:
Hi,

I'm a PFE from the Services organization and I'm suggesting some changes in order to allow this procedure to work with CentOS flawlessly and to make it clear that this procedure also works with a regular Active Directory domain.

Regards
Paulo

azurecla commented:
Hi __@ManaviS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

paulomarquesc committed:
Update active-directory-ds-admin-guide-join-rhel-linux-vm.md

Hi,

I'm a PFE from the Services organization and I'm suggesting some changes in order to allow this procedure to work with CentOS flawlessly and to make it clear that this procedure also works with a regular Active Directory domain.

Regards
Paulo
